Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/exportGroups.js

```diff
@@ -72,19 +72,17 @@ angular.module('virtoCommerce.exportModule')
                     setBreadcrumbs(node);
                 } else {
                     var selectedTypes = _.filter(blade.allGroups, function (x) { return x.groupName === node.groupName || (node.groupName === 'General' && !x.groupName); });
-                    console.log(selectedTypes[0]);
                     var exportDataRequest = {
                         exportTypeName: selectedTypes[0].typeName,
                         dataQuery: {
                             exportTypeName: selectedTypes[0].name + 'ExportDataQuery',
                             take: 10000
                         }
                     };
-                    console.log(exportDataRequest);
                     var newBlade = {
-                        id: 'pricelistExport',
-                        title: 'pricing.blades.exporter.priceListTitle',
-                        subtitle: 'pricing.blades.exporter.pricelistSubtitle',
+                        id: 'exportSettings',
+                        title: 'pricing.blades.exporter.' + selectedTypes[0].name +'Title',
```

Comment:
'pricing.blades.exporter.' + selectedTypes[0].name +'Title'
I think you need to use export module resource like 
`export.blades.exporter.title`
with some generic caption

Reply:
Maybe we do not need generic at all in this place? Please see last commit.

Reply:
Yes, we do need generic here at all. It should be common interface for all modules with one header.